cardano-testnet/src/Testnet/Property/Utils.hs

```diff
@@ -122,3 +127,18 @@ runInBackground act = void . H.evalM $ allocate (H.async act) cleanUp
 
 decodeEraUTxO :: (IsShelleyBasedEra era, MonadTest m) => ShelleyBasedEra era -> Aeson.Value -> m (UTxO era)
 decodeEraUTxO _ = H.jsonErrorFail . Aeson.fromJSON
+
+
+requireEon :: forall eon era m. Eon eon
```

Comment:
I think an haddock explaining what this function does would be beneficial. The type doesn't make obvious what's the actual era compared to the expected era.

Reply:
added

Reply:
Where do you use this function?

Reply:
Actually, I'm not using it anymore - I've refactored with `monoidForEraInEon`. I'll remove dead code.